### PR TITLE
Fix problem with service name (service_name option ignored)

### PR DESCRIPTION
**Pull request by otani88**

### CODE REVIEW

**Comment by otani88** [comment]
Fix version of proxysql, please.

**Comment by ernestas-poskus** [comment]
Thank you for the pull request, will look into version issue

**Comment by ernestas-poskus** [comment]
Version issue is fixed https://github.com/ernestas-poskus/chef-proxysql/pull/22

**Comment by otani88** [comment]
I see problems with installing mysql and another resources that don't depends on my changes.

File: test/integration/cookbooks/test-proxysql/recipes/default.rb [hunk 1]
```diff
@@ -6,7 +6,8 @@
   package 'mysql-server-5.6'
 end
 
-proxysql_service 'first' do
+proxysql_service 'first_service' do
+  service_name "proxysql-first"
```
Comment (ernestas-poskus) [inline test/integration/cookbooks/test-proxysql/recipes/default.rb:10]: this will be breaking change since proxysql is usually expected prefix

**Comment by ernestas-poskus** [comment]
> I see problems with installing mysql and another resources that don't depends on my changes.

it sometimes happens, I will try to prepare or use docker containers with mysql ready

**Comment by ernestas-poskus** [comment]
@mkryva rebase please

**Comment by ernestas-poskus** [comment]
@mkryva rebase please

**Pull request closed without merging**